tracing/providers/jaeger/src/main/java/io/helidon/tracing/providers/jaeger/JaegerTracerBuilder.java

```diff
@@ -335,13 +351,61 @@ public JaegerTracerBuilder trustedCertificates(io.helidon.common.configurable.Re
         return this;
     }
 
+    /**
+     * Span Processor type used.
+     *
+     * @param spanProcessorType to use
+     * @return updated builder
+     */
+    @ConfiguredOption("batch")
+    public JaegerTracerBuilder spanProcessorType(SpanProcessorType spanProcessorType) {
+        this.spanProcessorType = spanProcessorType;
+        return this;
+    }
+
+    /**
+     * Schedule Delay of exporter requests.
+     *
+     * @param scheduleDelayMillis timeout to use
+     * @return updated builder
+     */
+    @ConfiguredOption("PT5S")
+    public JaegerTracerBuilder scheduleDelay(Duration scheduleDelayMillis) {
```

Comment:
please check that the naming, defaults etc. are aligned both in types and in names - we should not have `millis` anywhere in the code.

Reply:
ah, yes. my bad! Fixed!